src/Textarea/index.js

```diff
@@ -111,45 +110,98 @@ StyledTextArea.defaultProps = {
 };
 
 // $FlowExpected
-const Textarea = React.forwardRef((props: Props, ref: Ref) => {
-  const {
-    size = SIZE_OPTIONS.NORMAL,
-    disabled,
-    resize = RESIZE_OPTIONS.VERTICAL,
-    dataTest,
-    spaceAfter,
-  } = props;
-
-  return (
-    <Field fullHeight={props.fullHeight} spaceAfter={spaceAfter}>
-      {props.label && (
-        <FormLabel filled={!!props.value} disabled={disabled}>
-          {props.label}
-        </FormLabel>
-      )}
-      <StyledTextArea
-        data-test={dataTest}
-        name={props.name}
-        value={props.value}
-        rows={props.rows}
-        size={size}
-        fullHeight={props.fullHeight}
-        disabled={disabled}
-        error={props.error}
-        placeholder={props.placeholder}
-        maxLength={props.maxLength}
-        onChange={props.onChange}
-        onFocus={props.onFocus}
-        onBlur={props.onBlur}
-        resize={resize}
-        tabIndex={props.tabIndex}
-        ref={ref}
-      />
-      {props.help && !props.error && <FormFeedback type="help">{props.help}</FormFeedback>}
-      {props.error && <FormFeedback type="error">{props.error}</FormFeedback>}
-    </Field>
-  );
-});
+const Textarea = React.forwardRef(
```

Comment:
Can you delete the suppress comments and use types for the `forwardRef`? 🙏 (Also in other form components)